wagtail_localize/segments/ingest.py

```diff
@@ -226,10 +227,23 @@ def ingest_segments(original_obj, translated_obj, src_locale, tgt_locale, segmen
         field_name, segment = segment.unwrap()
         segments_by_field_name[field_name].append(segment)
 
-    for field_name, field_segments in segments_by_field_name.items():
+    for field in get_translatable_fields(translated_obj.__class__):
+
+        field_name = field.field_name
+        field_segments = segments_by_field_name[field_name]
         field = translated_obj.__class__._meta.get_field(field_name)
 
-        if hasattr(field, "restore_translated_segments"):
+        if not field_segments:
+
+            if isinstance(field, (models.TextField, models.CharField)):
+                data = field.value_from_object(original_obj)
+            else:
+                # TODO implement for other types
+                continue
+
+            setattr(translated_obj, field_name, data)
```

Comment:
To synchronise a field with the source, you should register it as a ``SynchronisedField``. I don't think localize should try to be clever about fields that are not configured